extract_unique_sensors_01.py

```diff
@@ -62,19 +62,18 @@ def extract_all_sensors(sensors_data, sensor_name):
     return candidate_lines
 
 # Loop over candidate_lines and extract sensor names
-def splitting_list(sensors_list):
+def splitting_list(sensors_data = read_sensors(file_name, url)):
     """Splits the candidate_lines by dot and append that to sensors_list. It also removes the 
     empty lines.
     """
     sensors_list = []
     candidate_lines = sensors_list
```

Comment:
Can you explain what's going on here?

Reply:
Candidate_lines overwrites sensors_list, my mistake.